src/Umbraco.Cms.Integrations.Automation.Zapier/Migrations/ZapContentConfigTable.cs

```diff
@@ -67,6 +67,9 @@ public class ZapContentConfig
 
             [Column("WebHookUrl")] 
             public string WebHookUrl { get; set; }
```

Comment:
We should make sure `WebHookUrl` has a unique index set on it (as we're now using it for look-ups).
`ContentTypeName` should have one as well for similar reasons.

An example [here](https://github.com/umbraco/Forms/blob/v10/dev/src/Umbraco.Forms.Core/Persistence/Dtos/UserGroupFormSecurity.cs#L30) - though this isn't a unique index, but I think the attribute used here has options for that.

Reply:
Updated.